src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -80,6 +80,9 @@ struct Settings;
     M(UInt64, replicated_max_parallel_fetches_for_host, DEFAULT_COUNT_OF_HTTP_CONNECTIONS_PER_ENDPOINT, "Limit parallel fetches from endpoint (actually pool size).", 0) \
     M(UInt64, replicated_max_parallel_sends, 0, "Limit parallel sends.", 0) \
     M(UInt64, replicated_max_parallel_sends_for_table, 0, "Limit parallel sends for one table.", 0) \
+    M(Seconds, replicated_fetches_http_connection_timeout, 0, "HTTP connection timeout for part fetch requests. Inherited from default profile `http_connection_timeout` if not set explicitly.", 0) \
```

Comment:
Can someone please double-check whether this is true? I tried to re-read the code, and it seems that there is no way to actually change timeouts in a non-query context.

Reply:
It should be possible to change it in default profile (or in system profile if it is set).
I did not check it, though.